bower.json

```diff
@@ -26,19 +26,20 @@
   ],
   "dependencies": {
     "polymer": "^1.6.0",
-    "px-theme": "^3.0.0",
     "px-tooltip": "^1.0.0",
     "px-polymer-font-awesome": "^2.0.0",
     "iron-dropdown": "^1.5.4",
     "iron-a11y-keys": "^1.0.7"
   },
   "devDependencies": {
+    "px-theme": "^3.0.0",
     "px-spacing-design": "^1.0.0",
     "px-defaults-design": "^2.0.0",
```

Comment:
The main thing that I removed was px-starter-kit because it was importing more than we needed.